databusclient/client.py

```
@@ -1,721 +1,875 @@
+    debug: bool
+        controls whether output shold be printed to the console (stdout)
+    """
+
+    headers = {"X-API-KEY": f"{api_key}", "Content-Type": "application/json"}
+    data = json.dumps(dataid)
+    base = "/".join(dataid["@graph"][0]["@id"].split("/")[0:3])
+    api_uri = (
+        base
+        + f"/api/publish?verify-parts={str(verify_parts).lower()}&log-level={log_level.name}"
+    )
+    resp = requests.post(api_uri, data=data, headers=headers)
```

Ruff (0.14.2)

Probable use of `requests` call without timeout

(S113)